language/tools/test-generation/src/abstract_state.rs

```diff
@@ -106,6 +109,83 @@ impl AbstractValue {
     }
 }
 
+#[derive(Debug, Clone, PartialEq)]
+pub struct CallGraph {
+    calls: HashMap<FunctionHandleIndex, HashSet<FunctionHandleIndex>>,
```

Comment:
Yes, only in-module. But I don't think we need to worry even in the case where we allow cross-calls to other modules (since those will be represented as a function handle in the current module).

Reply:
right but then you move to the other module where the set of handles are different, right?
but it's all good for now

Reply:
ahhh I get what you're saying. Agreed that this isn't on the plate right now. (And, I don't think it should be too hard to update this to take this into account).